packages/permissions/src/components/branch-on-permissions/branch-on-permissions.tsx

```diff
@@ -51,6 +51,7 @@ const branchOnPermissions =
         }
         render={(isAuthorized) => {
           if (isAuthorized) {
+            // @ts-ignore Related to https://github.com/emotion-js/emotion/issues/3245
```

Comment:
With the latest changes in `main`, perhaps this ignore directive is not needed anymore. Could you please double check? 🙏